src/symbols.js

```diff
@@ -492,6 +492,7 @@ defineSymbol(math, main, rel, "\u2190", "\\gets");
 defineSymbol(math, main, rel, ">", "\\gt");
 defineSymbol(math, main, rel, "\u2208", "\\in");
 defineSymbol(math, main, rel, "\u2209", "\\notin");
+defineSymbol(math, main, rel, "\u0338", "\\not");
```

Comment:
The plain `/` is not as slanted as the combining version of the glyph.  I've updated the code so that `\not` is just a symbol and not a function.   buildHTML.js now does some post-process on `\not`s in `buildExpression`.